README.md

```diff
@@ -67,10 +67,21 @@ Launch the application with the following commands:
     ros2 launch hello_world_simulation empty_world.launch.py
     ```
 
+- *Running Simulation Application With Visualization*
```

Comment:
the README states the following below, "When running simulation applications, run command with gui:=true to run gazebo client for visualization". i think that is sufficient and succinct enough and don't really see the benefit of having a section just for getting the GUI